### PR TITLE
add stateful config and util methods

### PR DESCRIPTION
**Pull request by nlu90**

### CODE REVIEW

**Comment by nlu90** [comment]
@srkukarni added stateful.yaml into other schedulers. 
This stateful.yaml contains config for LocalFileSystemStorage. People can change it whenever necessary

File: heron/spi/src/java/com/twitter/heron/spi/common/Context.java [hunk 1]
```diff
@@ -278,4 +292,16 @@ public static String shellBinary(Config cfg) {
   public static String pythonInstanceBinary(Config cfg) {
     return cfg.getStringValue(Key.PYTHON_INSTANCE_BINARY);
   }
+
+  public static final Boolean isCleanStateCheckpoints(Config cfg) {
+    return cfg.getBooleanValue(Key.IS_CLEAN_STATEFUL_CHECKPOINTS);
+  }
+
+  @SuppressWarnings("unchecked")
+  public static final String stateBackendSandboxClassPath(Config cfg) {
```
Comment (billonahill) [inline heron/spi/src/java/com/twitter/heron/spi/common/Context.java]: Do we need this? We mostly refactored out the concept of "sandbox" by introducing config value wildcards.

Comment (nlu90) [inline heron/spi/src/java/com/twitter/heron/spi/common/Context.java]: @billonahill how about change the name to `statefulStorageClassPath` which corresponds to `Context.schedulerClassPath`?

File: heron/spi/src/java/com/twitter/heron/spi/common/Context.java [hunk 2]
```diff
@@ -278,4 +292,15 @@ public static String shellBinary(Config cfg) {
...
     return cfg.getStringValue(Key.PYTHON_INSTANCE_BINARY);
   }
+
+  public static final Boolean isCleanStateCheckpoints(Config cfg) {
+    return cfg.getBooleanValue(Key.IS_CLEAN_STATEFUL_CHECKPOINTS);
+  }
+
+  @SuppressWarnings("unchecked")
+  public static final String statefulStorageClassPath(Config cfg) {
+    Map<String, Object> backendConfig = (Map<String, Object>) cfg.get(Key.STATEFUL_STORAGE_CONF);
```
Comment (billonahill) [inline heron/spi/src/java/com/twitter/heron/spi/common/Context.java]: s/backendConfig/statefulStorageConfig

**Comment by nlu90** [comment]
@billonahill  updated

**Pull request merged**